src/dynamic_honey_badger/mod.rs

```diff
@@ -149,14 +149,14 @@ struct KeyGenState<N> {
     /// The key generation instance.
     key_gen: SyncKeyGen<N>,
     /// The change for which key generation is performed.
-    change: Change<N>,
-    /// The number of key generation messages received from each peer. At most _N + 1_ are
+    change: NodeChange<N>,
+    /// The number of key generation messages received from the candidate. At most _N + 1_ are
```

Comment:
Sorry, the other change in this comment on master should also stay. We're tracking the key generation message count for each peer now, instead of just the candidate.